### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by n-eiling**
We should add more CODEOWNERS

### CODE REVIEW

**Comment by stv0g** [comment]
Please dont get me wrong. But I think the codeowners file should refelct who has written / conteibuted to which areas of the code. Your change does not reflect this anymore. W.g. Felix has not conteibuted to VILLASnode at all till now.

I also believe some familiarity with the code is required for a proper review.

A review isnt just a procedural hurdle. It should improve the contribution in a meaningful way. And I havnt seen any comment or feedback in the reviews from Felix or Manuel in the past.

I know this is hard. It takes time to dig into someones elses code and understand the changes they are proposing and beeing able to procide meqningful feedback. But in my opinion, its the only way to keep a high code quality in the project.

I am happy to add more codeowners. But that requires that they also do thourough reviews and not just approve everything without actually reading the code.

**Comment by n-eiling** [comment]
Ok I agree with you. It's just not practical to have you review everything. I don't think it's really necessary for you to look at new node types for example.
I actually did a review with Felix in person. And you did no review for your last PR. So we both pressed the merge without waiting for requirements box but I at least did a review (even though it might not have been as thorough as yours). So the question really is: what should the requirements be? Having a review from a current codeowner on everything obviously doesn't work.

**Comment by stv0g** [comment]
How about we remove the requirement for an approval and rely on CODEOWNERS alone? I think this would unblock us as people could work alone on their area of the code. E.g. you could work on VILLASfpga without being blocked by a review. For changes which touch other parts of the code, we would still require a review as those changes potentially also affect others.

I believe for changes which are only affecting yourself, we can skip the review part. But as soon as your changes affect code of others it should be required. And those reviews should be done by those affected. E.g. Felix approving changes of code he never used is not optimal here.

I agree with you that I should not need to review all new node-types. For new code, the respective authors should add themselves directly to the CODEOWNERS. In that case, I do not need to review it. If GitHub still requires me to approve the change to the CODEOWNERS, I could do that quickly..

> I actually did a review with Felix in person. And you did no review for your last PR. 

I would be nice if you could do your review in the open, so others can participate and comment on it as well. Because for reviews done in private its really hard to reason and build trust.

> Having a review from a current codeowner on everything obviously doesn't work.

I think it works. I also try hard to do my reviews quickly. We just need more codeowners :D

>  So we both pressed the merge without waiting for requirements box

Yeah, the reviewing procedure aside. We must at least check that the pipelines pass :)

**Comment by n-eiling** [comment]
> How about we remove the requirement for an approval and rely on CODEOWNERS alone?

Yes that would resolve the issue for me. Can you change the setting?

I also agree with your other points.

**Comment by stv0g** [comment]
> Can you change the setting?

Yes I will do it.

Can you maybe also adjust this PR to only add yourself as the owner of the hwdef-parse script?
I think this was the only change in your last PR which required me as a reviewer.

**Comment by n-eiling** [comment]
finally got around doing this.

File: CODEOWNERS
Comment (stv0g) [inline CODEOWNERS:16]: I think this is redundant to line 21?

**Review by stv0g — CHANGES_REQUESTED** [review]
I think we also need to ping Jonathan again about the CI runner.. :(

**Comment by stv0g** [comment]
>> Can you change the setting?

> Yes I will do it.

Done :)

**Review by stv0g — APPROVED** [review]

**Pull request merged**
